test/ctype_internal_test.c

```diff
@@ -53,7 +58,10 @@ static struct {
     { '%', '%' },
     { '~', '~' },
     {   0,   0 },
-    { EOF, EOF }
+    { EOF, EOF },
+    { 333, 333 },
+    { -333, -333 },
+    { -128, -128 }
```

Comment:
Yeah, sorry I looked at the wrong branch :(

Those three should go.  The positive is too large for an unsigned char.
Commit amended.